### PR TITLE
Update contributors

### PR DESCRIPTION
**Pull request by clementbiron**
- Use local images to prevent image url changes.
- Use a `json` file to make contributors maintenance easier.
- Add missing contributors.

### CODE REVIEW

**Comment by clementbiron** [comment]
- [x] TG agreement
- [ ] POD agreement
- [ ] LL agreement
- [ ] CTO agreement
- [ ] GS agreement
- [ ] PC agreement

File: src/modules/OTA-api/data-components/Contributors.tsx [hunk 1]
```diff
@@ -203,25 +43,25 @@ const Contributors: React.FC<ContributorsProps> = React.memo(
       >
         {subtitle && <h4 className={classNames(s.contributors_subtitle)}>{subtitle}</h4>}
         <div className={s.contributors_items}>
-          {contributors.map(({ login, avatar_url, html_url }) => {
+          {shuffle(Object.entries(contributors)).map(([name, url]) => {
```
Comment (MattiSG) [inline src/modules/OTA-api/data-components/Contributors.tsx:46]: I would prefer we keep chronological order instead of shuffling 🙂

**Review by MattiSG — APPROVED** [review]
Nice!

Is the README still up to date, specifically the section mentioning “in order for the service to retrieve Github contributors”? I am not sure that part is still relevant now.

**Comment by MattiSG** [comment]
I added a commit to optimise images, as some were still uncompressed. Please make sure to squash that commit before merging, so we don't waste repository space by storing uncompressed images 🙂

**Review by martinratinaud — APPROVED** [review]

**Pull request merged**